security/pkg/stsservice/tokenmanager/google/tokenexchangeplugin.go

```diff
@@ -124,13 +129,23 @@ func (p *Plugin) useCachedToken() ([]byte, bool) {
 		return nil, false
 	}
 
+	var cacheHitCount uint64 = 0
+	p.mutex.Lock()
+	p.accessTokenCacheHit++
+	cacheHitCount = p.accessTokenCacheHit
+	p.mutex.Unlock()
+
 	token := v.(stsservice.TokenInfo)
 	remainingLife := time.Until(token.ExpireTime)
-	pluginLog.Infof("find a cached access token with remaining lifetime: %s", remainingLife.String())
+	if cacheHitCount%cacheHitDivisor == 0 {
+		pluginLog.Infof("find a cached access token with remaining lifetime: %s", remainingLife.String())
```

Comment:
Based on offline discussion, this will be lower to debug and we will make it log more frequent.

Reply:
I change cacheHitDivisor to 50, please let me know if you think 50 is still big.